docs/spec/manifest-v2-1.md

```diff
@@ -0,0 +1,146 @@
+# Image Manifest Version 2, Schema 1 
+
+This document outlines the format of of the V2 image manifest.  Image manifests
+describe the various constituents of a docker image.  Image manifests can be
+ serialized to JSON format with the following media types:
+
+Manifest Type  | Media Type
+------------- | -------------
+manifest  | "application/vnd.docker.distribution.manifest.v1+json"
+signed manifest  | "application/vnd.docker.distribution.manifest.v1+prettyjws"
+
+*Note that "application/json" will also be accepted for schema 1.*
+
+References: 
+ 
+ - [Proposal: JSON Registry API V2.1](https://github.com/docker/docker/issues/9015) 
+ - [Proposal: Provenance step 1 - Transform images for validation and verification](https://github.com/docker/docker/issues/8093)
+
+## *Manifest* Field Descriptions
+
+Manifest provides the base accessible fields for working with V2 image format
+ in the registry.
+
+- **`name`** *string*
+
+	name is the name of the image's repository
```

Comment:
@dmcgowan This decision really is made by the registry or manifest format, so I see it as a client implementation detail. This is something we will have to work out with the namespace work.